File: hammer-gui/src/main/java/dev/vini2003/hammer/gui/HGUI.java
package dev.vini2003.hammer.gui;

import dev.vini2003.hammer.gui.registry.common.HGUIEvents;
import dev.vini2003.hammer.gui.registry.common.HGUINetworking;
import net.fabricmc.api.ModInitializer;

public class HGUI implements ModInitializer {
	@Override
	public void onInitialize() {
		HGUIEvents.init();
		HGUINetworking.init();
	}
}

File: hammer-gui/src/main/kotlin/dev/vini2003/hammer/gui/HGUI.kt
package dev.vini2003.hammer.gui


import dev.vini2003.hammer.gui.registry.common.HGUIEvents
import dev.vini2003.hammer.gui.registry.common.HGUINetworking
import net.fabricmc.api.ModInitializer

object HGUI : ModInitializer {
	override fun onInitialize() {
		HGUINetworking.init()
		HGUIEvents.init()
	}
}